modules/arm_plugin/demos/coco_detection_android_demo/README.md

```diff
@@ -0,0 +1,130 @@
+      -DIE_EXTRA_MODULES="$WORK_DIR/openvino_contrib/modules" \
+      -DBUILD_java_api=ON \
+      -DCMAKE_TOOLCHAIN_FILE="$WORK_DIR/android-ndk-r20/build/cmake/android.toolchain.cmake" \
+      -DANDROID_ABI=arm64-v8a \
+      -DANDROID_PLATFORM=29 \
+      -DANDROID_STL=c++_shared \
+      -DENABLE_SAMPLES=OFF \
+      -DENABLE_OPENCV=OFF \
+      -DENABLE_CLDNN=OFF \
+      -DENABLE_VPU=OFF \
+      -DENABLE_GNA=OFF \
+      -DENABLE_MYRIAD=OFF \
+      -DENABLE_TESTS=OFF \
+      -DENABLE_GAPI_TESTS=OFF \
+      -DENABLE_BEH_TESTS=OFF \
+      -B "$WORK_DIR/openvino/build" \
+      -S "$WORK_DIR/openvino"
+make -C "$WORK_DIR/openvino/build" --jobs=$(nproc --all)
+```
+
+- OpenVINO binaries could be stripped to reduce their size:
+
+```bash
+$WORK_DIR/android-ndk-r20/toolchains/aarch64-linux-android-4.9/prebuilt/linux-x86_64/aarch64-linux-android/bin/strip $WORK_DIR/openvino/bin/aarch64/Release/lib/*.so
+```
+
+The built results are in `$WORK_DIR/openvino/bin/aarch64/Release/lib`. We will use them later.
+
+> Please confirm that your `plugins.xml` in `$WORK_DIR/openvino/bin/aarch64/Release/lib` contains plugin name `"CPU"` and `libarmPlugin.so` library file in `$WORK_DIR/openvino/bin/aarch64/Release/lib`
+
+### Import demo project on Android Studio
+
+In this step, we will import demo project to infer object detection.
+
+- Choose and download [Android Studio](https://developer.android.com/studio) on your PC.
+
+- Clone latest branch of OpenVINO Contrib.
+
+```bash
+git clone https://github.com/openvinotoolkit/openvino_contrib.git "$WORK_DIR/demo"
+```
+
+- Select "File -> Open", and import demo project in `"$WORK_DIR/demo/openvino_contrib/modules/java_api/demos/coco_detection_android_demo"`.
+
+- Copy libraries and model files to the corresponding folder.
+
+  1. Clone `"$WORK_DIR/openvino/bin/aarch64/Release/lib/inference_engine_java_api.jar"` to `app/libs` folder.
+  2. Clone `"$WORK_DIR/openvino/bin/aarch64/Release/lib/*.so"` and `"$WORK_DIR/android-ndk-r20/sources/cxx-stl/llvm-libc++/libs/arm64-v8a/libc++_shared.so"` to `"app/src/main/jniLibs/arm64-v8a"`
+  3. Clone `"$WORK_DIR/openvino/bin/aarch64/Release/lib/plugins.xml"` to `"app/src/main/assets"`
+  4. Download and convert model "ssdlite_mobilenet_v2" [or pelee-coco, efficientdet-d0-tf] with Open Model Zoo in following steps and copy `"$WORK_DIR/open_model_zoo/tools/downloader/intel/ssdlite_mobilenet_v2/FP32/ssdlite_mobilenet_v2.xml"`, `"$WORK_DIR/open_model_zoo/tools/downloader/intel/ssdlite_mobilenet_v2/FP32/ssdlite_mobilenet_v2.bin"` to `"app/src/main/assets"`
+
+```bash
+git clone --depth 1 https://github.com/openvinotoolkit/open_model_zoo "$WORK_DIR/open_model_zoo"
+cd "$WORK_DIR/open_model_zoo/tools/downloader"
+python3 -m pip install -r requirements.in
+omz_downloader  --name ssdlite_mobilenet_v2 --output_dir $WORK_DIR/open_model_zoo/tools/downloader
+omz_converter --name ssdlite_mobilenet_v2 --download_dir $WORK_DIR/open_model_zoo/tools/downloader --precision FP32
```

Comment:
I recommend download only FP16 model because it's just smaller but gives same accuracy and performance.

Reply:
Thanks, I have replace FP16 to FP32.